Makefile

```
@@ -1,42 +1,37 @@
-PYTHON=3.10
-BASENAME=$(shell basename $(CURDIR))
 PROFILE_NAME=simple-dataops-k8s
 
 ######################
 #   initialization   #
 ######################
-env:
-	conda create -y -n $(BASENAME) python=$(PYTHON)
-
-install-pdm:
-	@echo "Install pdm";\
+install-poetry:
+	@echo "Install poetry";\
 	if [ `command -v pip` ];\
-		then pip install pdm;\
+		then pip install poetry;\
 	else\
-		curl -sSL https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py | python3 -;\
+		curl -sSL https://install.python-poetry.org | python3 -;\
```

답변이 늦었네요 😅 
```suggestion
		curl -sSL https://install.python-poetry.org | python3 - --version 1.4.2;\
```
이렇게 바꾸면 될 것 같습니다!

---

반영했습니다~